lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tests for Progress Tab API in the Course Home API
+"""
+
+from datetime import datetime
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from lms.djangoapps.course_home_api.toggles import COURSE_HOME_MICROFRONTEND
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from openedx.core.djangoapps.user_api.preferences.api import set_user_preference
+from openedx.core.djangoapps.waffle_utils.testutils import override_waffle_flag
+from student.models import CourseEnrollment
+from student.tests.factories import UserFactory
+
+@override_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
+@ddt.ddt
+class ProgressTabTestViews(BaseCourseHomeTests):
+    """
+    Tests for the Progress Tab API
+    """
+    def setUp(self):
+        super().setUp()
+        self.url =  reverse('course-home-progress-tab', args=[self.course.id])
+        ContentTypeGatingConfig.objects.create(enabled=True, enabled_as_of=datetime(2017, 1, 1))
+
+    @ddt.data(CourseMode.AUDIT, CourseMode.VERIFIED)
```

Comment:
If we aren't expecting a difference due to enrollment mode, you don't need to test both. If there is a difference, explicitly check that.

I also know this is largely in progress so if you'll use this later, that's fine

Reply:
there will be a difference in the certificate data so I think i'll leave it in for that purpose!